Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -615,13 +615,20 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
     .filter(isTrue)
     .mapConst(0)
 
-    self.configureQualtrics = self.applicationLaunchOptionsProperty.signal.map { _ in
+    self.configureQualtrics = Signal.zip(
```

Comment:
Hmm, I made this zip because I didn't want this to emit again if the config updates, but perhaps we do want that if the user logs in during the app session?